index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="description" content="Social - a student project" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8" />
+    <title>Social - Log in</title>
+    <link rel="stylesheet" href="dist/css/index.css" />
+    <script src="../node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
+  </head>
+  <body class="container-fluid px-0">
+    <header class="bg-primary px-3">
+      <nav
+        id="navbar"
+        class="navbar navbar-expand d-flex justify-content-between container-sm"
+      >
+        <a class="navbar-brand" href="feed/index.html">Social</a>
+        <ul class="navbar-nav">
+          <!-- <li class="nav-item">
+            <a class="nav-link" href="profile/index.html">Profile</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="feed/index.html">Feed</a>
+          </li> -->
+        </ul>
+      </nav>
+    </header>
+    <main
+      class="mt-5 d-flex flex-column justify-content-center align-items-center"
+    >
+      <h1>Log in</h1>
+      <div
+        class="card p-3 rounded-0 mt-5 d-flex flex-column align-items-center"
+        style="width: 20rem"
+      >
+        <form action="/profile">
+          <div class="pt-2">
+            <label for="mail" class="form-label">E-mail</label>
+            <input
+              type="email"
+              class="form-control rounded-0"
+              id="mail"
+              required
+            />
+          </div>
+          <div class="pt-2">
+            <label for="password" class="form-label">Password</label>
+            <input
+              type="text"
+              class="form-control rounded-0"
+              id="password"
+              required
+              minlength="8"
+            />
+          </div>
+          <div
+            class="d-flex justify-content-center align-items-center flex-column"
+          >
+            <button class="btn btn-secondary mt-3 rounded-0" type="submit">
+              Log in
+            </button>
+          </div>
+        </form>
+        <span class="pt-2"> or </span>
+        <div>
+          <button
+            class="btn btn-secondary mt-3 rounded-0"
+            type="button"
+            data-bs-toggle="modal"
+            data-bs-target="#registerModal"
+          >
+            Register
+          </button>
+          <div class="modal fade" id="registerModal">
+            <div class="modal-dialog">
+              <div class="modal-content rounded-0">
+                <div class="modal-body d-flex justify-content-center">
+                  <div class="card p-3 rounded-0 my-5" style="width: 20rem">
```

Comment:
```suggestion
                  <div class="card p-3 rounded-0 my-5">
```

Although this may be used on the Bootstrap docs, inline styles are **strictly against best practice** in HTML/CSS. This is because it increases the number of places we have to look to find styling information, whereas centralising all the styling information in a CSS file allows for quick and easy changes.